packages/plasma-web/src/components/Notification/NotificationsStore.ts

```diff
@@ -0,0 +1,66 @@
+import { createStoreon, StoreonStore } from 'storeon';
+
+import { NotificationProps } from './Notification';
+
+export type NotificationItem = {
+    id: string;
+    isHiding?: boolean;
+};
+export type NotificationsState = {
```

Comment:
а может просто
```suggestion
export type NotificationsState = NotificationItem[]
```

или storeon'у именно объект нужен?